build/build.js

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+
+const doT = require('./doT.min')
```

Comment:
It does, but thought maybe I could avoid npm since the library is so small and has no dependencies. But eh, probably not worth fighting it.